Exiled.Events/Patches/Events/Warhead/Detonated.cs

```diff
@@ -29,6 +28,45 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
         {
             List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
 
+            Label continueLabel = generator.DefineLabel();
+            LocalBuilder ev = generator.DeclareLocal(typeof(DetonatingEventArgs));
+
+            newInstructions[0].WithLabels(continueLabel);
```

Comment:
Little thing, dont use `.WithLabels` in that case, use `instructions[0].labels.Add(label)`. 
It is better to use `.WithLabels` only for situations when you'll should keep your modified instruction and do something with that later